README.md

```diff
@@ -46,40 +53,65 @@ Usage: suffuse <options> [path path ...]
   -n="": volume name (OSX only)
   -t=false: create scratch directory as mount point
   -v=false: log at INFO level
-
-% suffuse -m /mnt $HOME &
-% ls -l /mnt
-%
 ```
 
-Development
-===========
+_Preparing a playground_
+```
+# Create a directory to play in.
+% mkdir -p ~/tmp/scratch
 
+# Fill a file with digits 1 to 10.
+% seq 1 10 > ~/tmp/scratch/seq.txt
 ```
-# Continuous testing
-bin/cc.sh
-# Docker container works somewhat
-bin/docker.sh
+
+_Mounting_
 ```
+# Create a directory that holds the mount.
+% mkdir ~/mnt
+
+# Mount a directory through suffuse.
+# The `&` runs suffuse in a separate process.
+% suffuse -m ~/mnt ~/tmp/scratch &
+[2] 9134
 
-Taster
-======
+# List the contents through the suffuse mount.
+% ls ~/mnt
+seq.txt
+```
 
+_Start playing_
 ```
-% seq 1 10 > /scratch/seq.txt
-% suffuse -m /mnt /scratch &
 # It's a 10 line file, one number to a line.
-% wc -l /mnt/seq.txt
-      10 /mnt/seq.txt
-# Via suffuse, a derived file: 4,6p is a sed command.
+% wc -l ~/mnt/seq.txt
+10 /home/user/mnt/seq.txt
+
+# Via suffuse, a derived file ending with #4,6p is a sed 
+# command executed on the actual file.
 % cat /mnt/seq.txt#4,6p
 4
 5
 6
+
 # Arbitrary sed commands, different sized files.
 % ls -l /mnt/seq.txt#5,10p
--rw-r--r--  1 paulp  wheel  13 Jun 30 11:57 /mnt/seq.txt#5,10p
+-rw-r--r--  1 user  user  13 Jun 30 11:57 /mnt/seq.txt#5,10p
+
 # These files are effectively indistinguishable from "real" files.
 % ls -l /mnt/seq.txt#1,3p
--rw-r--r--  1 paulp  wheel  6 Jun 30 11:57 /mnt/seq.txt#1,3p
+-rw-r--r--  1 user  user  6 Jun 30 11:57 /mnt/seq.txt#1,3p
+```
+
+_Kill the suffuse instance_
+```
+% kill 9134
```

Comment:
See where it says `[2] 9134` ? You can kill that with `kill 9134` or with `kill %2` which will almost always be `kill %1` - it's only higher if you already had background processes. When you have to type out pids you're doing it wrong.